content/en/docs/reference/score-spec-reference.md

```diff
@@ -92,29 +86,16 @@ resources:
-| `volume`      | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. |
-| `service`     | N/A                                                                                                                             | Translates to the module properties references. For example: `${modules.service-name.property-name}`.                           |
-| `workload`    | N/A                                                                                                                             | Reserved resource type. Its usage may lead to compatibility issues with future releases of [score-humanitec](https://github.com/score-spec/score-humanitec).|
-
+| Resource type | `score-compose`                                                                                                                 | `score-humanitec`                                                                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `environment` | Translates to the environment variables references. For example: `${PROPERTY-NAME}`.                                            | Translates to the application values references. For example: `${values.property-name}`.                                                                     |
+| `volume`      | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification.                              |
+| `service`     | N/A                                                                                                                             | Translates to the module properties references. For example: `${modules.service-name.property-name}`.                                                        |
+| `workload`    | N/A                                                                                                                             | Reserved resource type. Its usage may lead to compatibility issues with future releases of [score-humanitec](https://github.com/score-spec/score-humanitec). |
 
 ### Referencing resources
```

Comment:
This section should be updated as well, what do you think @ghen?

- Line 102: "Resources are declared in the containers section of the Score file and must map to the YAML structure defined in the resources section to properly resolve." - This sentence doesn't make total sense I think. Resources are declared in a dedicated `resources` section of the Score file.

- Line 107: "${resources.resource-name.property-name}" - This is no longer possible. Same for the example below.

Reply:
Line 102 - Resources section in the container specification has nothing to do with the resources section in the score file. This whole sentence should be removed.

Reply:
Line 110-113 - not sure what this block mean. We do not have it anywhere in the score file.